csharp/ql/src/experimental/ir/implementation/unaliased_ssa/Instruction.qll

```diff
@@ -742,7 +742,7 @@ class NoOpInstruction extends Instruction {
  * The `ReturnInstruction` for a function will have a control-flow successor edge to a block
  * containing the `ExitFunction` instruction for that function.
  *
- * There are two differet return instructions: `ReturnValueInstruction`, for returning a value from
+ * There are two different return instructions: `ReturnValueInstruction`, for returning a value from
```

Comment:
equivalent to https://github.com/github/codeql/pull/10825/commits/764a9e563f608c4c6d0c3545aef28d4bd5fb02c9#diff-0bf207ef7b91cba94e929ed8c56fcba31c81ed3c8605898b6150ebdfea9ee641R745